integration-tests/deployment/cs_test.go

```diff
@@ -472,3 +443,76 @@ func TestDeploy(t *testing.T) {
 		fmt.Print("JSON encoded TON state view:\n" + string(data))
 	})
 }
+
+func TestDeployMCMS(t *testing.T) {
```

Comment:
Let's try to structure MCMS deployments (changesets/ops/seq/tests) separate from CCIP - this is a distinct protocol that can exist without CCIP and not coupling the two when not needed will simplify our codebase.